packages/celotool/src/cmds/deploy/initial/oracle-fullnode.ts

```diff
@@ -0,0 +1,27 @@
+import { InitialArgv } from 'src/cmds/deploy/initial'
+import { installFullNodeChart } from 'src/lib/aks-fullnode'
+import {
+  addOracleMiddleware,
+  getAzureClusterConfig,
+  getOracleAzureContext,
+  OracleArgv,
+  switchToAzureContextCluster,
+} from 'src/lib/oracle'
+import yargs from 'yargs'
+
+export const command = 'oracle-fullnode'
+
+export const describe = 'deploy the oracle full-node(s) on an AKS cluster'
+
+type OracleFullNodeInitialArgv = InitialArgv & OracleArgv
+
+export const builder = (argv: yargs.Argv) => {
+  return addOracleMiddleware(argv)
+}
+
+export const handler = async (argv: OracleFullNodeInitialArgv) => {
+  const oracleAzureContext = getOracleAzureContext(argv.primary)
+  await switchToAzureContextCluster(argv.celoEnv, oracleAzureContext)
+  const clusterConfig = getAzureClusterConfig(oracleAzureContext)
+  await installFullNodeChart(argv.celoEnv, clusterConfig)
```

Comment:
What do you think about wrapping these three methods in a single call that returns the cluster config taking into account the context?
  

```
const oracleAzureContext = getOracleAzureContext(argv.primary)
await switchToAzureContextCluster(argv.celoEnv, oracleAzureContext)
const clusterConfig = getAzureClusterConfig(oracleAzureContext)
```

Reply:
I kind of like having the separate functions, mostly because it's only for the oracle full nodes, they're separate things, and I don't like relying on `argv.primary`. If we find ourselves needing to do this in more places then I'd be up for merging them into a more portable function